src/game.js

```diff
@@ -0,0 +1,117 @@
+import Tile from './tiles';
+
+class Game {
+  constructor(size) {
+    const tilesWin = [];
+    this.tilesWin = tilesWin;
+    this.length = size * size;
+   
+    this.bindMethods();
+  
+  }
+  getWinnerTiles() {
+    for (let i = 0; i < this.length - 1; i++) {
+      this.tilesWin[i] = i + 1;
```

Comment:
This way is fine, but just to remind, usually if we want to put a new value at the end of the array we use `Array.prototype.push()`
`this.tilesWin.push(i)`

Reply:
Good idea! Thanks!

Reply:
done